libs/module-to-standalone/admin/feature/src/lib/admin-feature.routes.ts

```diff
@@ -0,0 +1,18 @@
+import { Routes } from '@angular/router';
+
+export const ADMIN_ROUTES: Routes = [
+  {
+    path: '',
+    loadComponent: () =>
+      import('./dashboard/dashboard.component').then(
+        (m) => m.DashboardComponent,
```

Comment:
you can use the default keyword is you want to avoid the then close. But that's only a proposal, some people prefer it more explicit